config/prod.json

```diff
@@ -1,5 +1,5 @@
 {
   "googleClientId": "424501080111-b3bt4p1vo4gbo4m0573nvi4d49784bp8.apps.googleusercontent.com",
-  "leoUrlRoot": "https://leonardo.dsde-prod.broadinstitute.org",
+  "leoUrlRoot": "https://notebooks.firecloud.org",
```

Comment:
This should be a separate PR. That would allow me to merge the fix to prod while we discuss this PR.